style/base.css

```diff
@@ -231,3 +231,29 @@ ul.jp-notebook-job-output-formats-options {
 .jp-notebook-job-navigation-tab.active {
   border-block-end: 2px solid var(--jp-brand-color1);
 }
+
+
+.jp-collapsible-panel {
```

Comment:
Let's use the CSS class naming convention of `.jp-jobs-<Component>` so `jp-jobs-CollapsiblePanel`.

Reply:
Done